tests/queries/0_stateless/01833_tupleElement_formatting_fuzzer.sql

```diff
@@ -0,0 +1,2 @@
+select tupleElement(255, 100); -- { serverError 183 }
```

Comment:
Ok but it's quite unnatural that the exception is thrown from the function that formats AST. If server will not format AST, we won't get the exception here.

Maybe just format it as `tupleElement(255, 100)`?

Type checking is not the responsibility of AST formatter.

Reply:
>Type checking is not the responsibility of AST formatter.

Yeah, totally agree (I had bad feeling about this patch)

>Maybe just format it as tupleElement(255, 100)?

Sounds better - done.